sources/main.cpp

```diff
@@ -116,35 +116,43 @@ main(int argc, char* argv[]) {
     context.execute();
 
     // Traces must be re-initiliazed to append to current DynaflowLauncher log file as simulation had modified it
-    dfl::common::Log::init(options, config.outputDir(), continuePreviousLogFile);
```

Comment:
Maybe modify Log::init function since we no longer need the continuePreviousLogFile argument

Reply:
solved